Makefile

```diff
@@ -3,11 +3,13 @@ COVERAGE_DIR=${BUILD_DIR}/coverage
 BEATS=packetbeat topbeat filebeat winlogbeat metricbeat
 PROJECTS=libbeat ${BEATS}
 
-# Runs complete testsuites (unit, system, integration) for all beats,
-# with coverage and race detection.
+# Runs complete testsuites (unit, system, integration) for all beats with coverage and race detection.
+# Also it builds the docs and the generators
 .PHONY: testsuite
 testsuite:
 	$(foreach var,$(PROJECTS),$(MAKE) -C $(var) testsuite || exit 1;)
+	#$(MAKE) -C generate test
+	$(MAKE) docs
```

Comment:
I kept it in to not have to change the jenkins command and it is currently incompatible with 1.3. but I will just update 1.3 very soon, so this is not a problem. I removed it.